docs/source/explanation/additional_features/auto_configuration.rst

```diff
@@ -1,4 +1,37 @@
 Auto-configuration
 ==================
 
-Coming soon...
\ No newline at end of file
+Auto-configuration for a deep learning framework means the automatic finding of the most appropriate settings for the training parameters, based on the dataset and the specific task at hand.
+Auto-configuration can help to save time, ease the process of interaction with OTX CLI and give a better baseline for the given dataset.
+
+At this end, we developed a simple auto-configuration functionality to ease the process of training and validation utilizing our framework.
+Basically, to start the training and obtain a good baseline with the best trade-off between accuracy and speed we need to pass only a dataset in the right format without specifiing anything else:
+
+.. code-block::
+
```

Comment:
Auto-configuration also supports `otx build --train-data-roots <path_to_data_root>`

Reply:
Is there a reason to run `otx build` instead of `otx train` for users? If we call `otx build` it means `otx` will create a folder with all components, but the template (model) will be the middle one. So, if we use auto-config calling otx build is unnecessary, IMO. But, I will add it as a note mentioning that `otx build` can be called to create a folder.